### PR TITLE
IALERT-3158:  Have UI use new REST API for failed audit entries

### PR DESCRIPTION
**Pull request by psantos1113**
This PR makes slight changes to the Audit Page to display only failed audit entries.  It will utilize the new database tables created in a previous pull request.

Changes:
- Make request to api/audit/failed
- Remove the show only sent notifications checkbox
- Update the audit page description

### CODE REVIEW

File: component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryActions.java [hunk 1]
```diff
@@ -1,167 +1,44 @@
-/*
```
Comment (psantos1113) [inline component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryActions.java:1]: This class was renamed.  The code that is removed here is the old code.  The old accessor is named AuditEntryActionsLegacy.

File: component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryActionsLegacy.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
+/*
```
Comment (psantos1113) [inline component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryActionsLegacy.java:1]: This is the original actions class  AuditEntryActions.  There were no changes other than renaming the class.

File: component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryController.java [hunk 3]
```diff
@@ -1,79 +1,42 @@
-/*
```
Comment (psantos1113) [inline component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryController.java:1]: Renamed the AuditEntryController to AuditEntryControllerLegacy.  The contents of this class are the new Rest API endpoint.

File: component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryControllerLegacy.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
+/*
```
Comment (psantos1113) [inline component/src/main/java/com/synopsys/integration/alert/component/audit/web/AuditEntryControllerLegacy.java:1]: This is the old controller AuditEntryController.  The only change in this file is to rename the class.

File: ui/src/main/js/store/actions/audit.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ import * as HTTPErrorUtils from 'common/util/httpErrorUtilities';
 import { unauthorized } from 'store/actions/session';
 import HeaderUtilities from 'common/util/HeaderUtilities';
 
-const FETCH_URL = '/alert/api/audit';
+const FETCH_URL = '/alert/api/audit/failed';
```
Comment (bseifert14) [inline ui/src/main/js/store/actions/audit.js:6]: If this is for a failing request, we should label the variable to be more specific.  Something like FETCH_URL_FAIL or FETCH_URL_ERROR

Comment (psantos1113) [reply ui/src/main/js/store/actions/audit.js]: Updated

**Review by bseifert14 — APPROVED** [review]

**Pull request merged**